test/domain/relation/relation.spec.ts

```diff
@@ -0,0 +1,37 @@
+
+import { describe, expect, it } from 'vitest';
+import { CREATOR0, CREATOR1, RelationAlreadyExists, establish, getFollowers, getFollowing } from './_fixtures/relation.fixture';
+
+describe('domain/relation', () =>
+{
+    describe('relation', () =>
+    {
+        it('should establish a relation', async () =>
+        {
+            const relation = await establish(CREATOR1, CREATOR0);
+
+            expect(relation.id).toBeDefined();
+        });
+
+        it('should give an already exists exception', async () =>
+        {
+            const promise = establish(CREATOR0, CREATOR1);
+
+            expect(promise).rejects.toStrictEqual(new RelationAlreadyExists);
+        });
+
+        it('should retrieve a relation for a follower', async () =>
+        {
+            const relation = await getFollowers(CREATOR0);
+
+            expect(relation[0].follower?.id).toBe(CREATOR1);
```

Comment:
```suggestion
            const relations = await getFollowers(CREATOR0);

            expect(relations[0].follower?.id).toBe(CREATOR1);
```

Do you think a test with N=1 is sufficient here?